qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -356,6 +375,17 @@ def init_private_profile():
         assert private_profile.isOffTheRecord()
         private_profile.setter.init_profile()
 
+        _qute_scheme_handler.install(private_profile)
```

Comment:
I suppose the duplicated code here could be moved to a common `_init_profile(profile, private=True|False)` method - but again, happy to defer that until after the PR is in.